config/consts.js

```diff
@@ -3,14 +3,19 @@
  */
 
 const APP_INDEX_HTML = 'index.html';
-const APP_INDEX_JS = 'index.js';
+const APP_INDEX_JS = 'app.js';
```

Comment:
since you're changing `index.js` to `app.js`, why not also change the constant name from `APP_INDEX_JS` to just `APP_JS`?